src/ide/vscode/VscodeEdit.ts

```diff
@@ -7,12 +7,12 @@ import {
 } from "@cursorless/vscode-common";
 import type * as vscode from "vscode";
 
-export default function vscodeEdit(
+export default async function vscodeEdit(
   editor: vscode.TextEditor,
   callback: (editBuilder: TextEditorEdit) => void,
   options?: { undoStopBefore: boolean; undoStopAfter: boolean },
-): Thenable<boolean> {
-  return editor.edit((editBuilder) => {
+): Promise<boolean> {
```

Comment:
Why are we using `Promise` instead of `Thenable`?

Reply:
I don't think thenable is part of typescript? There is of course 3rd party, but I don't really see the upside instead of using regular promises?